apps/web/src/pages/quick-start/steps/DigestPreview.tsx

```diff
@@ -47,23 +56,49 @@ function RightSide() {
       <NavButton navigateTo={ROUTES.TEMPLATES} style={{ marginRight: '40px' }}>
         <ButtonText>Build a Workflow</ButtonText>
       </NavButton>
-      <NavButton navigateTo={ROUTES.TEMPLATES} variant={'gradient'}>
+      <NavButton pulse={true} navigateTo={ROUTES.TEMPLATES} variant={'gradient'}>
         <ButtonText>Try the Digest Playground</ButtonText>
       </NavButton>
     </>
   );
 }
 
+const DemoContainer = styled.div`
+  margin-left: 300px;
+
+  transition: margin-left 0.3s ease;
+
+  @media (max-width: 1750px) {
+     {
+      margin-left: 150px;
+    }
+  }
+
+  @media (max-width: 1470px) {
+     {
+      margin-left: 30px;
+    }
+  }
+
+  @media (max-width: 1300px) {
+     {
+      margin-left: 10px;
+    }
```

Comment:
Was not sure what will be the best solution for the resizing at the moment used @media will love to hear if some one knows any best practices on those cases.

Reply:
we should use either the breakpoints that Mantine provides: https://mantine.dev/theming/theme-object/
or we should define our own set of breakpoints, and also we should choose which way we want to implement it, like mobile-first and then desktop or vice-versa